<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>10</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Number 1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>10</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Number 2</string>
    </property>
   </widget>
   <widget class="QLabel" name="result_2">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>10</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Result</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="num1">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>40</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="num2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>40</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="result">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>40</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn0">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>70</y>
      <width>41</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn1">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>70</y>
      <width>41</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn2">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>70</y>
      <width>41</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn3">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>100</y>
      <width>41</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn4">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>100</y>
      <width>41</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn5">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>100</y>
      <width>41</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn6">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>130</y>
      <width>41</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn7">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>130</y>
      <width>41</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn8">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>130</y>
      <width>41</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn9">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>160</y>
      <width>41</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnADD">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>70</y>
      <width>31</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSUB">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>100</y>
      <width>31</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnMUL">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>130</y>
      <width>31</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnDIV">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>160</y>
      <width>31</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>/</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnEnter">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>180</y>
      <width>41</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Enter</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnClear">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>180</y>
      <width>41</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btncount">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>40</y>
      <width>51</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Count</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnreset">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>70</y>
      <width>51</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="countDisplay">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>110</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="operationDisplay">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>150</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
